### DISCUSSION TITLE
Discussion: What should it take to close an Issue

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#1756] [General]
Collecting all the Issues and around Magento and OpenMage over the years was a lot of work, and its one of our most valuable Assets.

We do not have any OKRs(Objective Key Results) we need to fulfill or Management pressuring us into keeping the numbers of Issues and PRs down.
With Labels we have a way to organize them somehow.
And even with low info provided, over Time most of the issues find someone who is able to understand/extend/solve them.

I would suggest to set the bar to close a ticket as high, maybe even higher then the bar to merge a change.

**Comment by tmotyl** [comment]
The issue is that the longer the queue of open issues (or pr) is the smaller the throuput and longer the cycle time is. 
Maintaining the list of PRs and issues is a time consuming job which is not easy to find volonteers for.

Its possible to reopen any closed issue.

Im ok with keeping issues open which have good description and steps to reproduce especially if they have a label „reproducible” meaning somebody else managed to validate it.

But i would close all issues which are not clear after asking the author to give details and having no feedback for 30 days.

We have no okr’s but already people are looking at numbers like number of open issues as a benchmark to compare with other repos/ systems.

**Comment by Flyingmana** [comment]
Do you have any numbers for the reduced throughput?
For OpenMage it seems quite the opposite.

The issue with reopening closed Issues is, that there is no filter to find out if an issue was actually solved or just closed because one single person did not understand it.


I think to follow the example of Magento with its 30 days policy is an unfortunate decision.
There are already many people who think its not worth to report a bug to Magento,2, because the issue is to small or they do not have the time to describe it properly.
And even then its possible, that they just get closed.
So if the majority wants to go this way, then its fine,we can put this into an rfc and then its the rule. But think of the impact this will have.


Just the most recent comment about how Magento2 is handling issues:
https://www.linkedin.com/posts/yehor-shytikov-20a24438_magento-magneto2-activity-6679229077850869760-Ctaq

**Comment by sreichel** [comment]
I just closed a few old ones without propper description ... all got labels that filter them is possible,

PLMK how to deal with issues that have no description for years like #269, This was one reason to add issue-templates, to have clear descriptions with reproduable steps ...
